eoscompanion/dummy_feed.py

```diff
@@ -32,192 +32,156 @@ def dummy_feed():
                 'sources': [
```

Comment:
>>'payload': [
I think that this list can also be removed

Reply:
Good catch, fixed.